src/site/_includes/components/AuthorInfo.js

```diff
@@ -73,7 +73,9 @@ module.exports = ({post, author, showSocialMedia = false}) => {
       class="w-author__info"
       style="display: flex; flex-direction: column; justify-content: center;"
     >
-      <cite class="w-author__name">${fullName}</cite>
+      <cite class="w-author__name">
+        <a href="/authors/${id}">${fullName}</a>
```

Comment:
Linking the author's name on the post card and the page itself changes the appearance. I think we should write some some CSS to preserve the previous look, but add a :hover state that doubles as the :focus state by combining the selectors:

```
:hover,
:focus {
  // probably let the items turn blue
}

Reply:
So this affects a posts page, I can change the card, but how should I address a page?

Reply:
Maybe on the card we don't have a hover state at all (since we have one big one for the card itself). But we would have a :focus state which underlines the text (similar to how the focus state for the card heading works).

On the post page I would have it work similar to the breadcrumbs where it's grey but gets an underline on hover. And a blue outline + grey underline on focus.

![image](https://user-images.githubusercontent.com/1066253/74389252-77581480-4db2-11ea-9984-e4b6980272b3.png)